lib/gitx/github.rb

```diff
@@ -103,12 +103,12 @@ def create_authorization
       password = ask_without_echo("Github password for #{username}: ")
       client = Octokit::Client.new(login: username, password: password)
       options = {
-        :scopes => ['repo'],
-        :note => github_client_name,
-        :note_url => CLIENT_URL
+        scopes: ['repo'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.